examples/jsm/loaders/MMDLoader.js

```diff
@@ -1133,17 +1134,18 @@ class MaterialBuilder {
 				 * MMDToonMaterial doesn't have ambient. Set it to emissive instead.
 				 * It'll be too bright if material has map texture so using coef 0.2.
 				 */
-			params.diffuse = new Color().fromArray( material.diffuse );
+			params.diffuse = new Color().setRGB(
+				material.diffuse[ 0 ],
+				material.diffuse[ 1 ],
+				material.diffuse[ 2 ],
+				SRGBColorSpace
+			);
```

Comment:
Why not `params.diffuse = new Color().setRGB( ...material.diffuse, SRGBColorSpace );` here too?

Reply:
The issue here is the 4th component in the diffuse array, otherwise it would be fine.